src/app/containers/BulletedList/index.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin, arabic } from '@bbc/gel-foundations/scripts';
+import BulletedListContainer from './index';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { orderedList } from './fixtures';
+
+describe('BulletedListContainer', () => {
+  describe('with data', () => {
```

Comment:
No biggy but I don't really see the need to have your tests in 2 nested describes. I don't see the need for even one describe block because you can clearly see in the test output the name of the file.

Reply:
Working on it